src/neptune/internal/backends/hosted_file_operations.py

```diff
@@ -380,7 +380,7 @@ def upload_raw_data(
         HTTPPaymentRequired.status_code,
     ):
         raise NeptuneLimitExceedException(reason=response.json().get("title", "Unknown reason"))
-    response.raise_for_status()
```

Comment:
Not needed as this is now handled in `ApiMethodWrapper.handle_neptune_http_errors`